app/helpers/component-argument.js

```diff
@@ -0,0 +1,25 @@
+import { helper } from '@ember/component/helper';
+import { isArray } from '@ember/array';
+
+/**
+ * Determine the type of the component argument for display
+ *
+ * @method componentArgumentDisplay
+ * @param {*} argument
+ * @return {*} The argument with the correct type for display
+ */
+export function componentArgumentDisplay([argument]) {
+  if (typeof argument === 'string') {
+    return `"${argument}"`;
+  } else if (argument === null) {
+    return 'null';
+  } else if (argument === undefined) {
+    return 'undefined';
+  } else if(isArray(argument) || typeof argument === 'object'){
+    return '...';
+  }
+
+  return argument;
+}
+
+export default helper(componentArgumentDisplay);
```

Comment:
trailing new line

Reply:
Does this really matter? If it's important, it should be enforced by the ember lint rules.

Reply:
It's in the `.editorconfig` in the repo. If you are using vscode, there is an extension for it.